xapian-applications/omega/configure.ac

```diff
@@ -286,6 +286,11 @@ PKG_CHECK_MODULES([LIBARCHIVE], [libarchive], [
     AC_DEFINE(HAVE_LIBARCHIVE, 1, [Define HAVE_LIBARCHIVE if the library is available])
     OMINDEX_MODULES="$OMINDEX_MODULES omindex_libarchive"],[ ])
 
+dnl check if libextractor is available.
+PKG_CHECK_MODULES([LIBEXTRACTOR], [libextractor], [
+AC_DEFINE(HAVE_LIBEXTRACTOR, 1, [Define HAVE_LIBEXTRACTOR if the library is available])
+OMINDEX_MODULES="$OMINDEX_MODULES omindex_libextractor"],[ ])
```

Comment:
This isn't indented like the similar constructs just above.

Reply:
I have changed this.